### PR TITLE
Hs issue 86

### PR DESCRIPTION
**Pull request by holgerschill**

### CODE REVIEW

**Comment by holgerschill** [comment]
Merge works with bnd configs.

File: build.gradle [hunk 1]
```diff
@@ -11,6 +11,7 @@ buildscript {
 	dependencies {
 		classpath "org.xtext:xtext-gradle-plugin:$versions.xtext_gradle_plugin"
 		classpath "io.spring.gradle:dependency-management-plugin:$versions.dependency_management_plugin"
+		classpath 'biz.aQute.bnd:biz.aQute.bnd.gradle:4.2.0'
```
Comment (cdietrich) [inline build.gradle]: i think it would be better to move the 4.2.0 to a contant in versions.gradle

File: gradle/manifest-gen.gradle [hunk 2]
```diff
@@ -12,18 +12,10 @@ else if (baseVersion.split('\\.').length == 4)
...
-	instruction 'Import-Package', "org.eclipse.xtext.xbase.lib.*;version=\"${version}\""
-	instruction 'Import-Package', '!org.eclipse.xtext.web.example.*'
-	instruction 'Import-Package', '!org.eclipse.xtend2.lib.*'
-	instruction 'Import-Package', '!org.eclipse.xtext.junit4.*'
-	instruction 'Import-Package', '!org.hamcrest.*'
-	instruction 'Import-Package', '!org.junit.*'
-}
+jar.bnd (
+	'Bundle-Version': qualifiedVersion,
+	'Import-Package': "org.apache.log4j.*;version='1.2.15',com.google.common.*;version='[14.0,22.0)',org.eclipse.core.runtime.*;version=!;common=!,org.eclipse.xtext.xbase.lib.*;version=\"${version}\",!org.eclipse.xtext.web.example.*,!org.eclipse.xtext.idea.example.*,!org.eclipse.xtend2.lib.*,!org.eclipse.xtext.junit4.*,!org.hamcrest.*,!org.junit.*"
```
Comment (cdietrich) [inline gradle/manifest-gen.gradle]: the range looks bad to be we work with guava 21 only. thus this should be 21 upto excluding 22

Comment (cdietrich) [inline gradle/manifest-gen.gradle]: am not sure if our log4j versions are consitent. what do we use in the bom?

Comment (holgerschill) [reply gradle/manifest-gen.gradle]: 1.2.17

File: build.gradle (hunk 1 above)
Comment (holgerschill) [reply build.gradle]: Done

File: gradle/manifest-gen.gradle (hunk 2 above)
Comment (holgerschill) [reply gradle/manifest-gen.gradle]: Done

Comment (holgerschill) [reply gradle/manifest-gen.gradle]: Done

**Pull request merged**